en/docs/wip/overriding-developer-portal-theme.md

```diff
@@ -0,0 +1,433 @@
+     Uses of resources in `login` folder will be discussed in ## Applying thems for tenant login pages section.
+     
+      ```
+      └──apim
+      │    └──css
+      │    │    └── custom.css
+      │    ├── defaultTheme.json
+      │    └── images
+      │          └── custom-logo.png
+      └──analytics
+      │     └── images
+      │          └── custom-logo.png
+      │          └── favicon.ico
+      └──login
+           └──css
+           │    └── custom.css
+           ├── loginTheme.json
+           └── images
+                 └── custom-logo.png
+                 └── favicon.ico
+      ```
+
+3.  Sign in to the WSO2 Admin Portal ([https://server-host:9443/admin](https://server-host:9443/admin)) with your tenant username (format `<username>@<domain>.com kim@testorg.com`) and password.
+
+4.  Expand the **Settings** menu, click **Upload Tenant Theme** and upload your ZIP file. 
+
+    ![Upload tenant theme]({{base_path}}/assets/img/Learn/upload-tenant-theme.png)
+
+5.  Access the API Developer Portal (`https://<server-host>:9443/devportal`) using your tenant username and password.
+
+    Note the new theme is applied.
+
+## Adding custom logo for the tenant
+
+In your tenant theme, you can refer to an image from the `defaultTheme.json` file as follows. The examples below uses the `custom-logo.png` image from the `sampleTheme.zip`. The image can be referred using one of the following URL patterns.
+
+The path format to the `custom-logo.png` image within the tenant domain theme is as follows:
+
+```js
+"logo": "/site/public/tenant_themes/<tenant-domain>/apim/images/custom-logo.png",
+```
+The path to the `custom-logo.png` image within the `kim@testorg.com` tenant domain theme is as follows:
+
+```js
+"logo": "/site/public/tenant_themes/kim@testorg.com/apim/images/custom-logo.png",
+```
+
+The following defines the logo image from an external URL.
+
+```js
+"logo": "https://dummyimage.com/208x19/66aad1/ffffff&text=testlogo",
+```
+
+## Applying CSS rules to change the look and feel
+
+If you prefer to change the styling using CSS rules, you can use the `custom.css` file. The above sample theme also has a custom CSS file. In the CSS file, let's change the top header background color to black. 
+
+Note that we have injected IDs into the dom elements. You can use them to apply CSS rules. However, be aware about the dynamically generated CSS class names. These class names have a number suffix which changes from version to version. Therefore, it is recommended not to use them for styling purposes. 
+
+The CSS file is referenced in the `defaultTheme.json` in the following manner. It is not a must to replace the `<tenant-domain> ` in the following line, as at runtime the Developer Portal will automatically replace it with the current tenant domain.
+
+```js
+   "tenantCustomCss": "/site/public/tenant_themes/<tenant-domain>/apim/css/custom.css",
+```
+
+## Content of defaultTheme.json
+
+```js
+{
+  "themes": {
+    "light": {
```

Comment:
These two-lines should be removed. Unless theming won't work. Please update it in the sampleTheme.zip file as well.

Reply:
@msm1992 , were you able to address this change?